content/specific-skill-success-criteria/basic-data-analysis/_index.md

```diff
@@ -5,4 +5,151 @@ title: 'Assessment: Basic data analysis'
 topic_needs_review: true
 ---
 
-TODO
\ No newline at end of file
+All students must understand all of the following concepts:
+
+- Statistical concepts:  Mean, Median and Mode
+- Statistical concept:  IQR (Interquartile range)
+- Statistical concepts:  Standard deviation and Variance
+- Statistical concepts:  Correlation and p-value (Probability value) and Null-hypothesis vs. Alternative-hypothesis
+- Statistical concepts:  Standard deviation and Standard error
+
+### Statistical concepts: Mean, Median and Mode
+
+#### Mean (Average value):
+```
+Calculating the mean (average value):
+
+list_of_speed_data = [87, 94, 78, 77, 85, 86]
+
+mean = (87+94+78+77+85+86) = 507/6 = 84.5
+```
+
+
+#### Median (Middle value):
+```
+Calculating the median (middle value):
+
+list_of_speed_data = [100, 200, 300, 400, 500, 600]
+
+When we have an even amount of numbers we need to calculate the mean (average) of the two middle 
+numbers, in this case 300 and 400.  So the mean (average) of 300 and 400 is (300+400)/2 = 350.
+Therefore the mean (middle) value is:  median = 350
+
+
+When we have an odd amount of numbers we need to divide the total amount of numbers by 2 and then
+round up to the nearest integer.  For instance:
+
+list_of_speed_data = [100, 200, 300, 400, 500, 600, 700]
+
+There are seven numbers in the list, 7/2 = 3.5, if we round up to the nearest integer we get 4.  
+Therefore, the 4th number in the list will be the median (middle) value:  median = 400
+```
+
+
+#### Mode (Most frequent value):
+```
+The mode is the value which occurs the most in a set.  So if you had a set [2, 2, 2, 3] then the
+mode would be 2 since 2 is the value which occurs most frequently in the set.
+```
+
+
+
+### Statistical concepts: Interquartile range (IQR)
+1st quartile:  Middle number between the smallest number and the median of the dataset
```

Comment:
Given a list of numbers, the learner must be able to calculate the 1st, 2nd and 3rd quartile by hand or with simple code (no built in functions)